### PR TITLE
ci: use npm install instead of ci

### PR DESCRIPTION
**Pull request by michael-siek**
<< Describe the changes >>

Closes issue: #<< ticket number >>

### CODE REVIEW

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -25,7 +25,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-      - run: npm ci
+        # There is no `package-lock.json` file
+      - run: npm install
```
Comment (Zidious) [inline .github/workflows/create-release.yml:29]: if the package json has deps, we should add the package-lock onto the root

**Review by Zidious — DISMISSED** [review]
Minor thing

File: .github/workflows/create-release.yml [hunk 2]
```diff
@@ -25,7 +25,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-      - run: npm ci
+        # There is no `package-lock.json` file
```
Comment (dbowling) [inline .github/workflows/create-release.yml:28]: Why is this [different from the pip integration](https://github.com/dequelabs/axe-devtools-pip/blob/develop/package-lock.json)?

**Review by dbowling — CHANGES_REQUESTED** [review]
Quick question.

File: .github/workflows/create-release.yml (hunk 2 above)
Comment (michael-siek) [reply .github/workflows/create-release.yml:28]: https://github.com/dequelabs/axe-core-gems/pull/174

Comment (dbowling) [reply .github/workflows/create-release.yml:28]: Gotcha. This way actually makes _more_ sense.

File: .github/workflows/create-release.yml (hunk 1 above)
Comment (dbowling) [reply .github/workflows/create-release.yml:29]: Sounds like something that probably should have a SOP.

**Review by dbowling — APPROVED** [review]
Probably a retro topic in this. I'm good, but leave it to you to decide what to do with Gabe's open request.

**Pull request merged**